templates/web/fixmystreet.com/about/open311-api-info.html

```diff
@@ -0,0 +1,154 @@
+  <li>Find everything you need to know in the complete tech spec for the Open311 endpoint here: https://github.com/mysociety/fixmystreet/wiki/Open311-FMS---Complete-Spec</li>
+  <li>Once you've got a functional Open311 testing endpoint setup, email us the URL and API key and we’ll configure our FixMyStreet staging site to send appropriate FixMyStreet reports to that endpoint. From here you can inspect the data you receive and make adjustments until you’re happy.</li>
+  <li>You can then implement the spec we have given you access to, this includes</li>
+    <ul>
+      <li>Receiving reports (required)</li>
+      <li>Opening and closing reports (optional)</li>
+      <li>Listing services / Categories including extra questions (required)</li>
+      <li>Sending and receiving updates (optional)</li>
+      <li>Option to pull reports into the Open311 endpoint, this refers to reports not originally made in FixMyStreet </li>
+    </ul>
+  <li>Once you're satisfied it’s working and you have a fully compliant endpoint, email support@fixmystreet.com and we’ll update the live site to point at your live endpoint.</li>
+</ul>
+
+<p><strong>Please note:</strong> Once set up, the Open311 endpoint will override the list of categories visible on the FixMyStreet.com site and app according to those used within the connected backend system. Any existing FixMyStreet categories will be removed UNLESS you still want some additional categories to remain as email only (please let us know this beforehand).</p>
+
+</dd>
+
+<dt>Frequently asked questions</dt>
+
+<dl>
+<dt>Is it genuinely free?</dt>
+<dd>Yes, Open311 is a free technology. <a href= "https://www.open311.org/learn/"> Read more about it on the Open311 website.</a></dd> 
+
+<dt>What if we can’t implement this ourselves due to time or resource constraints?</dt>
+<dd>We can build and maintain the integration for you (carries an annual fee). Please contact us if you’d like to discuss this at support@fixmystreet.com.</dd> 
+
+<dt>Can we integrate with FixMyStreet in a different way?</dt>
+<dd>Yes, you can, but Open311 is a standardised way of sharing information, and provides the foundations should you ever want to use FixMyStreet Pro in the future.</dd> 
+</dl>
+
+<dt>Limitations</dt>
+
+<dd>
+<p>Open311 fetches a list of acceptable types of report as dictated by you, the council, so that the receiving service (in this case, FixMyStreet) can display them as options for the report-maker to choose from, sends reports into a backend system of your choice and enables you to close or update reports accordingly.</p>
+<p>What it doesn’t do is the next steps that may be required to further refine the reporting process, reduce unnecessary contact or help triage reports. This is where you might want to consider using FixMyStreet Pro - a more advanced version of FixMyStreet, which seeks to eliminate the significant costs councils incur from duplicate reporting, unnecessary contact and failure demand.</p>
+
+<table>
```

Comment:
If you change this to `<table class="dashboard-ranking-table">` it'll colour it nicely.

Reply:
Good to know!